frontend/src/components/common/form/AutoCompleteText.tsx

```diff
@@ -0,0 +1,72 @@
+import './AutoCompleteText.scss';
+
+import React from 'react';
+import { Form } from 'react-bootstrap';
+import { useState } from 'react';
+import { DisplayError } from './DisplayError';
+import { useFormikContext } from 'formik';
+
+export const AutoCompleteText: React.FC<any> = ({ field, options, placeholder }) => {
+  let items: any[] = [];
+
+  const { setFieldValue } = useFormikContext<any>();
+
+  options.forEach((option: { label: any; value: any }) => {
+    items.push(option);
+  });
+
+  const [suggestions, setSuggestions] = useState<any>([]);
+  const [text, setText] = useState<any>('');
+
+  const onTextChanged = (e: any) => {
+    const val = e.target.value;
+    let dynamicSuggestions: any[] = [];
+    if (val.length > 0) {
+      const regex = new RegExp(`^${val}`, 'i');
```

Comment:
I'm not sure we necessarily want the start of the string, my preference would be just a general match.

Reply:
fixed!